tasks/time-series/time-series-forecasting/XGB/Readme.md

```diff
@@ -0,0 +1,22 @@
+# Project Description
+
+Irish electricity consumption forecasting for the next 7-days using XGBoost.
+
+**Data**
+
+Data contains electricity consumption measurements for 15-min intervals from March, 2011 to February 2013.
+
+Source : [data.gov.ie](https://data.gov.ie/dataset/energy-consumption-gas-and-electricity-civic-offices-2009-2012/resource/6091c604-8c94-4b44-ac52-c1694e83d746)
+
+**Requirements**
+* xgboost
+* scikit-learn
+* matplotlib
+
+
+
+**Example run**
+
+Usage: python main.py --url {url} --output_file {output_file} --predicted_date {predicted_date}
```

Comment:
hi @melodiCyb, thanks a lot for your contribution. Could you please update the example with mlflow usage.
A working example would be great.

Please take a reference of MLflow quick start guide.
https://mlflow.org/docs/latest/quickstart.html#running-mlflow-projects

Reply:
updated mlflow usage